dynatademand/schemas/request/body/create_line_item.json

```diff
@@ -45,12 +45,7 @@
     "deliveryType": {
       "type": "string",
       "default": "BALANCED",
-      "description": "The plan on how responses will flow into the survey.",
-      "enum": [
```

Comment:
Why delete this?

Reply:
It is been replaced with string and validated for the input. The enum values are no longer being used in the implementation.